docs/manual/getting_started.rst

```diff
@@ -173,21 +190,23 @@ This step will copies everything to:
 At this point, you are ready to use Eclipse Cyclone DDS in your
 projects.
 
-**Note:** The default build type is a release build with debug
-information included (RelWithDebInfo). This is a convenient type of
-build to use from applications because of a good mix between performance
-and still being able to debug things. If you'd rather have a Debug or
-pure Release build, set ``CMAKE_BUILD_TYPE`` accordingly.
+**Note:** The default build type is a release build that includes debugging 
+information (RelWithDebInfo). 
+
+This is a convenient type of build to use from applications because 
+of a good mix between performance and still being able to debug things. 
+If you'd rather have a Debug or pure Release build, 
+set ``CMAKE_BUILD_TYPE`` accordingly.
 
 If you want to contribute to Cyclone DDS, please refer to `Appendix I
 Contributing to Eclipse Cyclone DDS for
 Windows. <#appendix-i-contributing-to-eclipse-cyclone-dds>`__
 
 Installation with product installer for Windows
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+'''''''''''''''''''''''''''''''''''''''''''''''
 
-The Cyclone DDS also provides a product installer, this may be easier
-than installing it from GitHub.
+The Cyclone DDS also provides a product installer, a more simple method 
+of installation rather than installing it from GitHub.
```

Comment:
I disagree with this reformulation. The previous statement boiled down to:
> Using the installer is easier in **some** environments

The reformulation corresponds to:
> Using the installer is easier in **all** environments

I don't think this is the impression that we want to provide to all developers who might be new to Cyclone.